### PR TITLE
Changed background and menu scripts into ES6 modules, added utils.mjs module

### PR DESCRIPTION
**Pull request by BenCrulis**
Scripts loaded as ES6 modules enable "use strict" so it is no longer necessary to declare "use strict" in these. background.js is also now in strict mode as a consequence.
This enables the use of static imports, which are cleaner that other methods of imports. utils.mjs is already imported in background.js and menu.js. Call functions from the imported utils module using `utils.myFunction(...)`. Other modules can also be added in the manifest and imported the same way.
Content scripts probably still require some workaround to be allowed to import from utils.mjs as they might not be recognized as ES6 modules, but perhaps it will work using dynamic imports.

utils.mjs contains the code snippet from issue https://github.com/tournesol-app/tournesol-chrome-extension/issues/6#issuecomment-830774469 for cookie handling for now.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     }
   ],
   "web_accessible_resources": [
-	  "rate_now_icon.png"
+	  "rate_now_icon.png",
+	  "lib.js"
```
Comment (lfaucon) [inline manifest.json:35]: I don't see this file. Either you forgot to `git add` it or maybe this should be replaced by `utils.mjs`

**Review by lfaucon — CHANGES_REQUESTED** [review]
Looks great! 

Does this work also from the content_scripts scripts? 

See comment above for requested change

File: manifest.json (hunk 1 above)
Comment (BenCrulis) [reply manifest.json]: Oops, I changed the name of the file at the last minute, fixed it.

**Comment by BenCrulis** [comment]
> Does this work also from the content_scripts scripts?
It does not seem to work using neither static or dynamic imports, at least on Firefox.
I'm looking for workarounds but I fear this will prevent the use of the chrome API anyway.

File: manifest.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   "content_scripts": [
     {
       "matches": ["https://*.youtube.com/*"],
-      "js": ["addTournesolRecommendations.js"],
```
Comment (lfaucon) [inline manifest.json:22]: This should also be done for the second content script (the one adding the rate-later button)

**Review by lfaucon — APPROVED** [review]
Nice. I think it is great merging this change as it is. Of course there is more refactoring coming up (mostly moving duplicated code to dry code in utils.js, thanks to this change)

**Pull request merged**